### PR TITLE
Create the_biological_bulletin.csl

### PR DESCRIPTION
**Pull request by nicolewebster**
handmade style to fit the Biological Bulletin journal literature cited requirements (http://www.biolbull.org/site/misc/ifora.xhtml#references)

### CODE REVIEW

**Comment by adam3smith** [comment]
The only thing wrong is the filename -- it should be the-biological-bulletin.csl (hyphens, not underscores).

We'd also ask you to put the template and documentation link back in, please. Looks great otherwise.

File: the_biological_bulletin.csl [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    <title>The Biological Bulletin</title>
+    <title-short>Biol.Bull.</title-short>
+    <id>http://www.zotero.org/styles/the-biological-bulletin</id>
+    <link href="http://www.zotero.org/styles/the-biological-bulletin" rel="self"/>
+    <author>
+      <name>Nicole Webster</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+    <issnl>1939-8697</issnl>
```
Comment (adam3smith) [inline the_biological_bulletin.csl]: oh, also we don't really use issnl -- just go with issn (or eissn, whichever applies).

Comment (rmzelle) [inline the_biological_bulletin.csl]: Yes, please use:

```
<issn>0006-3185</issn>
<eissn>1939-8697</eissn>
```

ISSN-L wasn't a bad idea, but it hasn't really seen much uptake, and most publishers only show ISSN and eISSN, so we focus on those two as well.

**Comment by rmzelle** [comment]
Sorry, but can you remove the extra quote mark that you just introduced?

`<link href=""http://www.biolbull.org/site/misc/ifora.xhtml#references" rel="documentation"/>` should be `<link href="http://www.biolbull.org/site/misc/ifora.xhtml#references" rel="documentation"/>`.

**Pull request closed without merging**